diff --git a/generator-build/index.js b/generator-build/index.js
index 94cff69cee29a619e8f725da3da9b7ff93884d96..0006c00855bfe38afbea296465719229be8c0755 100644
--- a/generator-build/index.js
+++ b/generator-build/index.js
@@ -125,7 +125,7 @@ var require_windows = __commonJS({
       return checkPathExt(path10, options);
     }
     function isexe(path10, options, cb) {
-      fs8.stat(path10, function(er, stat) {
+      fs8.stat(path10, function (er, stat) {
         cb(er, er ? false : checkStat(stat, path10, options));
       });
     }
@@ -143,7 +143,7 @@ var require_mode = __commonJS({
     isexe.sync = sync;
     var fs8 = require("fs");
     function isexe(path10, options, cb) {
-      fs8.stat(path10, function(er, stat) {
+      fs8.stat(path10, function (er, stat) {
         cb(er, er ? false : checkStat(stat, options));
       });
     }
@@ -191,8 +191,8 @@ var require_isexe = __commonJS({
         if (typeof Promise !== "function") {
           throw new TypeError("callback not provided");
         }
-        return new Promise(function(resolve, reject) {
-          isexe(path10, options || {}, function(er, is) {
+        return new Promise(function (resolve, reject) {
+          isexe(path10, options || {}, function (er, is) {
             if (er) {
               reject(er);
             } else {
@@ -201,7 +201,7 @@ var require_isexe = __commonJS({
           });
         });
       }
-      core(path10, options || {}, function(er, is) {
+      core(path10, options || {}, function (er, is) {
         if (er) {
           if (er.code === "EACCES" || options && options.ignoreErrors) {
             er = null;
@@ -240,7 +240,7 @@ var require_which = __commonJS({
         // windows always checks the cwd first
         ...isWindows ? [process.cwd()] : [],
         ...(opt.path || process.env.PATH || /* istanbul ignore next: very unusual */
-        "").split(colon)
+          "").split(colon)
       ];
       const pathExtExe = isWindows ? opt.pathExt || process.env.PATHEXT || ".EXE;.CMD;.BAT;.COM" : "";
       const pathExt = isWindows ? pathExtExe.split(colon) : [""];
@@ -537,7 +537,7 @@ var require_enoent = __commonJS({
         return;
       }
       const originalEmit = cp.emit;
-      cp.emit = function(name, arg1) {
+      cp.emit = function (name, arg1) {
         if (name === "exit") {
           const err = verifyENOENT(arg1, parsed, "spawn");
           if (err) {
@@ -682,7 +682,7 @@ var require_onetime = __commonJS({
       let returnValue;
       let callCount = 0;
       const functionName = function_.displayName || function_.name || "<anonymous>";
-      const onetime2 = function(...arguments_) {
+      const onetime2 = function (...arguments_) {
         calledFunctions.set(onetime2, ++callCount);
         if (callCount === 1) {
           returnValue = function_.apply(this, arguments_);
@@ -994,12 +994,12 @@ var require_realtime = __commonJS({
     "use strict";
     Object.defineProperty(exports2, "__esModule", { value: true });
     exports2.SIGRTMAX = exports2.getRealtimeSignals = void 0;
-    var getRealtimeSignals = function() {
+    var getRealtimeSignals = function () {
       const length = SIGRTMAX - SIGRTMIN + 1;
       return Array.from({ length }, getRealtimeSignal);
     };
     exports2.getRealtimeSignals = getRealtimeSignals;
-    var getRealtimeSignal = function(value, index) {
+    var getRealtimeSignal = function (value, index) {
       return {
         name: `SIGRT${index + 1}`,
         number: SIGRTMIN + index,
@@ -1023,13 +1023,13 @@ var require_signals = __commonJS({
     var _os = require("os");
     var _core = require_core();
     var _realtime = require_realtime();
-    var getSignals = function() {
+    var getSignals = function () {
       const realtimeSignals = (0, _realtime.getRealtimeSignals)();
       const signals = [..._core.SIGNALS, ...realtimeSignals].map(normalizeSignal);
       return signals;
     };
     exports2.getSignals = getSignals;
-    var normalizeSignal = function({
+    var normalizeSignal = function ({
       name,
       number: defaultNumber,
       description,
@@ -1056,11 +1056,11 @@ var require_main = __commonJS({
     var _os = require("os");
     var _signals = require_signals();
     var _realtime = require_realtime();
-    var getSignalsByName = function() {
+    var getSignalsByName = function () {
       const signals = (0, _signals.getSignals)();
       return signals.reduce(getSignalByName, {});
     };
-    var getSignalByName = function(signalByNameMemo, { name, number, description, supported, action, forced, standard }) {
+    var getSignalByName = function (signalByNameMemo, { name, number, description, supported, action, forced, standard }) {
       return {
         ...signalByNameMemo,
         [name]: { name, number, description, supported, action, forced, standard }
@@ -1068,13 +1068,13 @@ var require_main = __commonJS({
     };
     var signalsByName = getSignalsByName();
     exports2.signalsByName = signalsByName;
-    var getSignalsByNumber = function() {
+    var getSignalsByNumber = function () {
       const signals = (0, _signals.getSignals)();
       const length = _realtime.SIGRTMAX + 1;
       const signalsA = Array.from({ length }, (value, number) => getSignalByNumber(number, signals));
       return Object.assign({}, ...signalsA);
     };
-    var getSignalByNumber = function(number, signals) {
+    var getSignalByNumber = function (number, signals) {
       const signal = findSignalByNumber(number, signals);
       if (signal === void 0) {
         return {};
@@ -1092,7 +1092,7 @@ var require_main = __commonJS({
         }
       };
     };
-    var findSignalByNumber = function(number, signals) {
+    var findSignalByNumber = function (number, signals) {
       const signal = signals.find(({ name }) => _os.constants.signals[name] === number);
       if (signal !== void 0) {
         return signal;
@@ -1267,12 +1267,12 @@ var require_signal_exit = __commonJS({
   "../../node_modules/.pnpm/signal-exit@3.0.7/node_modules/signal-exit/index.js"(exports2, module2) {
     "use strict";
     var process4 = global.process;
-    var processOk = function(process5) {
+    var processOk = function (process5) {
       return process5 && typeof process5 === "object" && typeof process5.removeListener === "function" && typeof process5.emit === "function" && typeof process5.reallyExit === "function" && typeof process5.listeners === "function" && typeof process5.kill === "function" && typeof process5.pid === "number" && typeof process5.on === "function";
     };
     if (!processOk(process4)) {
-      module2.exports = function() {
-        return function() {
+      module2.exports = function () {
+        return function () {
         };
       };
     } else {
@@ -1294,9 +1294,9 @@ var require_signal_exit = __commonJS({
         emitter.setMaxListeners(Infinity);
         emitter.infinite = true;
       }
-      module2.exports = function(cb, opts) {
+      module2.exports = function (cb, opts) {
         if (!processOk(global.process)) {
-          return function() {
+          return function () {
           };
         }
         assert.equal(typeof cb, "function", "a callback must be provided for exit handler");
@@ -1307,7 +1307,7 @@ var require_signal_exit = __commonJS({
         if (opts && opts.alwaysLast) {
           ev = "afterexit";
         }
-        var remove = function() {
+        var remove = function () {
           emitter.removeListener(ev, cb);
           if (emitter.listeners("exit").length === 0 && emitter.listeners("afterexit").length === 0) {
             unload();
@@ -1321,7 +1321,7 @@ var require_signal_exit = __commonJS({
           return;
         }
         loaded = false;
-        signals.forEach(function(sig) {
+        signals.forEach(function (sig) {
           try {
             process4.removeListener(sig, sigListeners[sig]);
           } catch (er) {
@@ -1340,7 +1340,7 @@ var require_signal_exit = __commonJS({
         emitter.emit(event, code, signal);
       };
       sigListeners = {};
-      signals.forEach(function(sig) {
+      signals.forEach(function (sig) {
         sigListeners[sig] = function listener() {
           if (!processOk(global.process)) {
             return;
@@ -1357,7 +1357,7 @@ var require_signal_exit = __commonJS({
           }
         };
       });
-      module2.exports.signals = function() {
+      module2.exports.signals = function () {
         return signals;
       };
       loaded = false;
@@ -1367,7 +1367,7 @@ var require_signal_exit = __commonJS({
         }
         loaded = true;
         emitter.count += 1;
-        signals = signals.filter(function(sig) {
+        signals = signals.filter(function (sig) {
           try {
             process4.on(sig, sigListeners[sig]);
             return true;
@@ -1385,7 +1385,7 @@ var require_signal_exit = __commonJS({
           return;
         }
         process4.exitCode = code || /* istanbul ignore next */
-        0;
+          0;
         emit("exit", process4.exitCode, null);
         emit("afterexit", process4.exitCode, null);
         originalProcessReallyExit.call(process4, process4.exitCode);
@@ -1630,7 +1630,7 @@ var require_merge_stream = __commonJS({
   "../../node_modules/.pnpm/merge-stream@2.0.0/node_modules/merge-stream/index.js"(exports2, module2) {
     "use strict";
     var { PassThrough } = require("stream");
-    module2.exports = function() {
+    module2.exports = function () {
       var sources = [];
       var output = new PassThrough({ objectMode: true });
       output.setMaxListeners(0);
@@ -1654,7 +1654,7 @@ var require_merge_stream = __commonJS({
         return sources.length == 0;
       }
       function remove(source) {
-        sources = sources.filter(function(it) {
+        sources = sources.filter(function (it) {
           return it !== source;
         });
         if (!sources.length && output.readable) {
@@ -2380,7 +2380,7 @@ var require_js_tokens = __commonJS({
       value: true
     });
     exports2.default = /((['"])(?:(?!\2|\\).|\\(?:\r\n|[\s\S]))*(\2)?|`(?:[^`\\$]|\\[\s\S]|\$(?!\{)|\$\{(?:[^{}]|\{[^}]*\}?)*\}?)*(`)?)|(\/\/.*)|(\/\*(?:[^*]|\*(?!\/))*(\*\/)?)|(\/(?!\*)(?:\[(?:(?![\]\\]).|\\.)*\]|(?![\/\]\\]).|\\.)+\/(?:(?!\s*(?:\b|[\u0080-\uFFFF$\\'"~({]|[+\-!](?!=)|\.?\d))|[gmiyus]{1,6}\b(?![\u0080-\uFFFF$\\]|\s*(?:[+\-*%&|^<>!=?({]|\/(?![\/*])))))|(0[xX][\da-fA-F]+|0[oO][0-7]+|0[bB][01]+|(?:\d*\.\d+|\d+\.?)(?:[eE][+-]?\d+)?)|((?!\d)(?:(?!\s)[$\w\u0080-\uFFFF]|\\u[\da-fA-F]{4}|\\u\{[\da-fA-F]+\})+)|(--|\+\+|&&|\|\||=>|\.{3}|(?:[+\-\/%&|^]|\*{1,2}|<{1,2}|>{1,3}|!=?|={1,2})=?|[?~.,:;[\](){}])|(\s+)|(^$|[\s\S])/g;
-    exports2.matchToToken = function(match) {
+    exports2.matchToToken = function (match) {
       var token = { type: "invalid", value: match[0], closed: void 0 };
       if (match[1]) token.type = "string", token.closed = !!(match[3] || match[4]);
       else if (match[5]) token.type = "comment";
@@ -2515,49 +2515,49 @@ var require_lib = __commonJS({
     });
     Object.defineProperty(exports2, "isIdentifierChar", {
       enumerable: true,
-      get: function() {
+      get: function () {
         return _identifier.isIdentifierChar;
       }
     });
     Object.defineProperty(exports2, "isIdentifierName", {
       enumerable: true,
-      get: function() {
+      get: function () {
         return _identifier.isIdentifierName;
       }
     });
     Object.defineProperty(exports2, "isIdentifierStart", {
       enumerable: true,
-      get: function() {
+      get: function () {
         return _identifier.isIdentifierStart;
       }
     });
     Object.defineProperty(exports2, "isKeyword", {
       enumerable: true,
-      get: function() {
+      get: function () {
         return _keyword.isKeyword;
       }
     });
     Object.defineProperty(exports2, "isReservedWord", {
       enumerable: true,
-      get: function() {
+      get: function () {
         return _keyword.isReservedWord;
       }
     });
     Object.defineProperty(exports2, "isStrictBindOnlyReservedWord", {
       enumerable: true,
-      get: function() {
+      get: function () {
         return _keyword.isStrictBindOnlyReservedWord;
       }
     });
     Object.defineProperty(exports2, "isStrictBindReservedWord", {
       enumerable: true,
-      get: function() {
+      get: function () {
         return _keyword.isStrictBindReservedWord;
       }
     });
     Object.defineProperty(exports2, "isStrictReservedWord", {
       enumerable: true,
-      get: function() {
+      get: function () {
         return _keyword.isStrictReservedWord;
       }
     });
@@ -2571,7 +2571,7 @@ var require_escape_string_regexp = __commonJS({
   "../../node_modules/.pnpm/escape-string-regexp@1.0.5/node_modules/escape-string-regexp/index.js"(exports2, module2) {
     "use strict";
     var matchOperatorsRe = /[|\\{}()[\]^$+*?.]/g;
-    module2.exports = function(str) {
+    module2.exports = function (str) {
       if (typeof str !== "string") {
         throw new TypeError("Expected a string");
       }
@@ -2788,7 +2788,7 @@ var require_conversions = __commonJS({
     var channels;
     var labels;
     var model;
-    convert.rgb.hsl = function(rgb) {
+    convert.rgb.hsl = function (rgb) {
       var r = rgb[0] / 255;
       var g = rgb[1] / 255;
       var b = rgb[2] / 255;
@@ -2821,7 +2821,7 @@ var require_conversions = __commonJS({
       }
       return [h, s * 100, l * 100];
     };
-    convert.rgb.hsv = function(rgb) {
+    convert.rgb.hsv = function (rgb) {
       var rdif;
       var gdif;
       var bdif;
@@ -2832,7 +2832,7 @@ var require_conversions = __commonJS({
       var b = rgb[2] / 255;
       var v = Math.max(r, g, b);
       var diff = v - Math.min(r, g, b);
-      var diffc = function(c) {
+      var diffc = function (c) {
         return (v - c) / 6 / diff + 1 / 2;
       };
       if (diff === 0) {
@@ -2861,7 +2861,7 @@ var require_conversions = __commonJS({
         v * 100
       ];
     };
-    convert.rgb.hwb = function(rgb) {
+    convert.rgb.hwb = function (rgb) {
       var r = rgb[0];
       var g = rgb[1];
       var b = rgb[2];
@@ -2870,7 +2870,7 @@ var require_conversions = __commonJS({
       b = 1 - 1 / 255 * Math.max(r, Math.max(g, b));
       return [h, w * 100, b * 100];
     };
-    convert.rgb.cmyk = function(rgb) {
+    convert.rgb.cmyk = function (rgb) {
       var r = rgb[0] / 255;
       var g = rgb[1] / 255;
       var b = rgb[2] / 255;
@@ -2887,7 +2887,7 @@ var require_conversions = __commonJS({
     function comparativeDistance(x, y) {
       return Math.pow(x[0] - y[0], 2) + Math.pow(x[1] - y[1], 2) + Math.pow(x[2] - y[2], 2);
     }
-    convert.rgb.keyword = function(rgb) {
+    convert.rgb.keyword = function (rgb) {
       var reversed = reverseKeywords[rgb];
       if (reversed) {
         return reversed;
@@ -2906,10 +2906,10 @@ var require_conversions = __commonJS({
       }
       return currentClosestKeyword;
     };
-    convert.keyword.rgb = function(keyword) {
+    convert.keyword.rgb = function (keyword) {
       return cssKeywords[keyword];
     };
-    convert.rgb.xyz = function(rgb) {
+    convert.rgb.xyz = function (rgb) {
       var r = rgb[0] / 255;
       var g = rgb[1] / 255;
       var b = rgb[2] / 255;
@@ -2921,7 +2921,7 @@ var require_conversions = __commonJS({
       var z = r * 0.0193 + g * 0.1192 + b * 0.9505;
       return [x * 100, y * 100, z * 100];
     };
-    convert.rgb.lab = function(rgb) {
+    convert.rgb.lab = function (rgb) {
       var xyz = convert.rgb.xyz(rgb);
       var x = xyz[0];
       var y = xyz[1];
@@ -2940,7 +2940,7 @@ var require_conversions = __commonJS({
       b = 200 * (y - z);
       return [l, a, b];
     };
-    convert.hsl.rgb = function(hsl) {
+    convert.hsl.rgb = function (hsl) {
       var h = hsl[0] / 360;
       var s = hsl[1] / 100;
       var l = hsl[2] / 100;
@@ -2981,7 +2981,7 @@ var require_conversions = __commonJS({
       }
       return rgb;
     };
-    convert.hsl.hsv = function(hsl) {
+    convert.hsl.hsv = function (hsl) {
       var h = hsl[0];
       var s = hsl[1] / 100;
       var l = hsl[2] / 100;
@@ -2996,7 +2996,7 @@ var require_conversions = __commonJS({
       sv = l === 0 ? 2 * smin / (lmin + smin) : 2 * s / (l + s);
       return [h, sv * 100, v * 100];
     };
-    convert.hsv.rgb = function(hsv) {
+    convert.hsv.rgb = function (hsv) {
       var h = hsv[0] / 60;
       var s = hsv[1] / 100;
       var v = hsv[2] / 100;
@@ -3021,7 +3021,7 @@ var require_conversions = __commonJS({
           return [v, p, q];
       }
     };
-    convert.hsv.hsl = function(hsv) {
+    convert.hsv.hsl = function (hsv) {
       var h = hsv[0];
       var s = hsv[1] / 100;
       var v = hsv[2] / 100;
@@ -3037,7 +3037,7 @@ var require_conversions = __commonJS({
       l /= 2;
       return [h, sl * 100, l * 100];
     };
-    convert.hwb.rgb = function(hwb) {
+    convert.hwb.rgb = function (hwb) {
       var h = hwb[0] / 360;
       var wh = hwb[1] / 100;
       var bl = hwb[2] / 100;
@@ -3096,7 +3096,7 @@ var require_conversions = __commonJS({
       }
       return [r * 255, g * 255, b * 255];
     };
-    convert.cmyk.rgb = function(cmyk) {
+    convert.cmyk.rgb = function (cmyk) {
       var c = cmyk[0] / 100;
       var m = cmyk[1] / 100;
       var y = cmyk[2] / 100;
@@ -3109,7 +3109,7 @@ var require_conversions = __commonJS({
       b = 1 - Math.min(1, y * (1 - k) + k);
       return [r * 255, g * 255, b * 255];
     };
-    convert.xyz.rgb = function(xyz) {
+    convert.xyz.rgb = function (xyz) {
       var x = xyz[0] / 100;
       var y = xyz[1] / 100;
       var z = xyz[2] / 100;
@@ -3127,7 +3127,7 @@ var require_conversions = __commonJS({
       b = Math.min(Math.max(0, b), 1);
       return [r * 255, g * 255, b * 255];
     };
-    convert.xyz.lab = function(xyz) {
+    convert.xyz.lab = function (xyz) {
       var x = xyz[0];
       var y = xyz[1];
       var z = xyz[2];
@@ -3145,7 +3145,7 @@ var require_conversions = __commonJS({
       b = 200 * (y - z);
       return [l, a, b];
     };
-    convert.lab.xyz = function(lab) {
+    convert.lab.xyz = function (lab) {
       var l = lab[0];
       var a = lab[1];
       var b = lab[2];
@@ -3166,7 +3166,7 @@ var require_conversions = __commonJS({
       z *= 108.883;
       return [x, y, z];
     };
-    convert.lab.lch = function(lab) {
+    convert.lab.lch = function (lab) {
       var l = lab[0];
       var a = lab[1];
       var b = lab[2];
@@ -3181,7 +3181,7 @@ var require_conversions = __commonJS({
       c = Math.sqrt(a * a + b * b);
       return [l, c, h];
     };
-    convert.lch.lab = function(lch) {
+    convert.lch.lab = function (lch) {
       var l = lch[0];
       var c = lch[1];
       var h = lch[2];
@@ -3193,7 +3193,7 @@ var require_conversions = __commonJS({
       b = c * Math.sin(hr);
       return [l, a, b];
     };
-    convert.rgb.ansi16 = function(args) {
+    convert.rgb.ansi16 = function (args) {
       var r = args[0];
       var g = args[1];
       var b = args[2];
@@ -3208,10 +3208,10 @@ var require_conversions = __commonJS({
       }
       return ansi;
     };
-    convert.hsv.ansi16 = function(args) {
+    convert.hsv.ansi16 = function (args) {
       return convert.rgb.ansi16(convert.hsv.rgb(args), args[2]);
     };
-    convert.rgb.ansi256 = function(args) {
+    convert.rgb.ansi256 = function (args) {
       var r = args[0];
       var g = args[1];
       var b = args[2];
@@ -3227,7 +3227,7 @@ var require_conversions = __commonJS({
       var ansi = 16 + 36 * Math.round(r / 255 * 5) + 6 * Math.round(g / 255 * 5) + Math.round(b / 255 * 5);
       return ansi;
     };
-    convert.ansi16.rgb = function(args) {
+    convert.ansi16.rgb = function (args) {
       var color = args % 10;
       if (color === 0 || color === 7) {
         if (args > 50) {
@@ -3242,7 +3242,7 @@ var require_conversions = __commonJS({
       var b = (color >> 2 & 1) * mult * 255;
       return [r, g, b];
     };
-    convert.ansi256.rgb = function(args) {
+    convert.ansi256.rgb = function (args) {
       if (args >= 232) {
         var c = (args - 232) * 10 + 8;
         return [c, c, c];
@@ -3254,19 +3254,19 @@ var require_conversions = __commonJS({
       var b = rem % 6 / 5 * 255;
       return [r, g, b];
     };
-    convert.rgb.hex = function(args) {
+    convert.rgb.hex = function (args) {
       var integer = ((Math.round(args[0]) & 255) << 16) + ((Math.round(args[1]) & 255) << 8) + (Math.round(args[2]) & 255);
       var string = integer.toString(16).toUpperCase();
       return "000000".substring(string.length) + string;
     };
-    convert.hex.rgb = function(args) {
+    convert.hex.rgb = function (args) {
       var match = args.toString(16).match(/[a-f0-9]{6}|[a-f0-9]{3}/i);
       if (!match) {
         return [0, 0, 0];
       }
       var colorString = match[0];
       if (match[0].length === 3) {
-        colorString = colorString.split("").map(function(char) {
+        colorString = colorString.split("").map(function (char) {
           return char + char;
         }).join("");
       }
@@ -3276,7 +3276,7 @@ var require_conversions = __commonJS({
       var b = integer & 255;
       return [r, g, b];
     };
-    convert.rgb.hcg = function(rgb) {
+    convert.rgb.hcg = function (rgb) {
       var r = rgb[0] / 255;
       var g = rgb[1] / 255;
       var b = rgb[2] / 255;
@@ -3303,7 +3303,7 @@ var require_conversions = __commonJS({
       hue %= 1;
       return [hue * 360, chroma * 100, grayscale * 100];
     };
-    convert.hsl.hcg = function(hsl) {
+    convert.hsl.hcg = function (hsl) {
       var s = hsl[1] / 100;
       var l = hsl[2] / 100;
       var c = 1;
@@ -3318,7 +3318,7 @@ var require_conversions = __commonJS({
       }
       return [hsl[0], c * 100, f * 100];
     };
-    convert.hsv.hcg = function(hsv) {
+    convert.hsv.hcg = function (hsv) {
       var s = hsv[1] / 100;
       var v = hsv[2] / 100;
       var c = s * v;
@@ -3328,7 +3328,7 @@ var require_conversions = __commonJS({
       }
       return [hsv[0], c * 100, f * 100];
     };
-    convert.hcg.rgb = function(hcg) {
+    convert.hcg.rgb = function (hcg) {
       var h = hcg[0] / 360;
       var c = hcg[1] / 100;
       var g = hcg[2] / 100;
@@ -3378,7 +3378,7 @@ var require_conversions = __commonJS({
         (c * pure[2] + mg) * 255
       ];
     };
-    convert.hcg.hsv = function(hcg) {
+    convert.hcg.hsv = function (hcg) {
       var c = hcg[1] / 100;
       var g = hcg[2] / 100;
       var v = c + g * (1 - c);
@@ -3388,7 +3388,7 @@ var require_conversions = __commonJS({
       }
       return [hcg[0], f * 100, v * 100];
     };
-    convert.hcg.hsl = function(hcg) {
+    convert.hcg.hsl = function (hcg) {
       var c = hcg[1] / 100;
       var g = hcg[2] / 100;
       var l = g * (1 - c) + 0.5 * c;
@@ -3400,13 +3400,13 @@ var require_conversions = __commonJS({
       }
       return [hcg[0], s * 100, l * 100];
     };
-    convert.hcg.hwb = function(hcg) {
+    convert.hcg.hwb = function (hcg) {
       var c = hcg[1] / 100;
       var g = hcg[2] / 100;
       var v = c + g * (1 - c);
       return [hcg[0], (v - c) * 100, (1 - v) * 100];
     };
-    convert.hwb.hcg = function(hwb) {
+    convert.hwb.hcg = function (hwb) {
       var w = hwb[1] / 100;
       var b = hwb[2] / 100;
       var v = 1 - b;
@@ -3417,34 +3417,34 @@ var require_conversions = __commonJS({
       }
       return [hwb[0], c * 100, g * 100];
     };
-    convert.apple.rgb = function(apple) {
+    convert.apple.rgb = function (apple) {
       return [apple[0] / 65535 * 255, apple[1] / 65535 * 255, apple[2] / 65535 * 255];
     };
-    convert.rgb.apple = function(rgb) {
+    convert.rgb.apple = function (rgb) {
       return [rgb[0] / 255 * 65535, rgb[1] / 255 * 65535, rgb[2] / 255 * 65535];
     };
-    convert.gray.rgb = function(args) {
+    convert.gray.rgb = function (args) {
       return [args[0] / 100 * 255, args[0] / 100 * 255, args[0] / 100 * 255];
     };
-    convert.gray.hsl = convert.gray.hsv = function(args) {
+    convert.gray.hsl = convert.gray.hsv = function (args) {
       return [0, 0, args[0]];
     };
-    convert.gray.hwb = function(gray2) {
+    convert.gray.hwb = function (gray2) {
       return [0, 100, gray2[0]];
     };
-    convert.gray.cmyk = function(gray2) {
+    convert.gray.cmyk = function (gray2) {
       return [0, 0, 0, gray2[0]];
     };
-    convert.gray.lab = function(gray2) {
+    convert.gray.lab = function (gray2) {
       return [gray2[0], 0, 0];
     };
-    convert.gray.hex = function(gray2) {
+    convert.gray.hex = function (gray2) {
       var val = Math.round(gray2[0] / 100 * 255) & 255;
       var integer = (val << 16) + (val << 8) + val;
       var string = integer.toString(16).toUpperCase();
       return "000000".substring(string.length) + string;
     };
-    convert.rgb.gray = function(rgb) {
+    convert.rgb.gray = function (rgb) {
       var val = (rgb[0] + rgb[1] + rgb[2]) / 3;
       return [val / 255 * 100];
     };
@@ -3489,7 +3489,7 @@ var require_route = __commonJS({
       return graph;
     }
     function link(from, to) {
-      return function(args) {
+      return function (args) {
         return to(from(args));
       };
     }
@@ -3505,7 +3505,7 @@ var require_route = __commonJS({
       fn.conversion = path10;
       return fn;
     }
-    module2.exports = function(fromModel) {
+    module2.exports = function (fromModel) {
       var graph = deriveBFS(fromModel);
       var conversion = {};
       var models = Object.keys(graph);
@@ -3531,7 +3531,7 @@ var require_color_convert = __commonJS({
     var convert = {};
     var models = Object.keys(conversions);
     function wrapRaw(fn) {
-      var wrappedFn = function(args) {
+      var wrappedFn = function (args) {
         if (args === void 0 || args === null) {
           return args;
         }
@@ -3546,7 +3546,7 @@ var require_color_convert = __commonJS({
       return wrappedFn;
     }
     function wrapRounded(fn) {
-      var wrappedFn = function(args) {
+      var wrappedFn = function (args) {
         if (args === void 0 || args === null) {
           return args;
         }
@@ -3566,13 +3566,13 @@ var require_color_convert = __commonJS({
       }
       return wrappedFn;
     }
-    models.forEach(function(fromModel) {
+    models.forEach(function (fromModel) {
       convert[fromModel] = {};
       Object.defineProperty(convert[fromModel], "channels", { value: conversions[fromModel].channels });
       Object.defineProperty(convert[fromModel], "labels", { value: conversions[fromModel].labels });
       var routes = route(fromModel);
       var routeModels = Object.keys(routes);
-      routeModels.forEach(function(toModel) {
+      routeModels.forEach(function (toModel) {
         var fn = routes[toModel];
         convert[fromModel][toModel] = wrapRounded(fn);
         convert[fromModel][toModel].raw = wrapRaw(fn);
@@ -3587,15 +3587,15 @@ var require_ansi_styles = __commonJS({
   "../../node_modules/.pnpm/ansi-styles@3.2.1/node_modules/ansi-styles/index.js"(exports2, module2) {
     "use strict";
     var colorConvert = require_color_convert();
-    var wrapAnsi16 = (fn, offset) => function() {
+    var wrapAnsi16 = (fn, offset) => function () {
       const code = fn.apply(colorConvert, arguments);
       return `\x1B[${code + offset}m`;
     };
-    var wrapAnsi256 = (fn, offset) => function() {
+    var wrapAnsi256 = (fn, offset) => function () {
       const code = fn.apply(colorConvert, arguments);
       return `\x1B[${38 + offset};5;${code}m`;
     };
-    var wrapAnsi16m = (fn, offset) => function() {
+    var wrapAnsi16m = (fn, offset) => function () {
       const rgb = fn.apply(colorConvert, arguments);
       return `\x1B[${38 + offset};2;${rgb[0]};${rgb[1]};${rgb[2]}m`;
     };
@@ -3967,7 +3967,7 @@ var require_chalk = __commonJS({
       if (!this || !(this instanceof Chalk) || this.template) {
         const chalk = {};
         applyOptions(chalk, options);
-        chalk.template = function() {
+        chalk.template = function () {
           const args = [].slice.call(arguments);
           return chalkTag.apply(null, [chalk.template].concat(args));
         };
@@ -4003,7 +4003,7 @@ var require_chalk = __commonJS({
       styles[model] = {
         get() {
           const level = this.level;
-          return function() {
+          return function () {
             const open = ansiStyles.color[levelMapping[level]][model].apply(null, arguments);
             const codes = {
               open,
@@ -4024,7 +4024,7 @@ var require_chalk = __commonJS({
       styles[bgModel] = {
         get() {
           const level = this.level;
-          return function() {
+          return function () {
             const open = ansiStyles.bgColor[levelMapping[level]][model].apply(null, arguments);
             const codes = {
               open,
@@ -4039,7 +4039,7 @@ var require_chalk = __commonJS({
     var proto = Object.defineProperties(() => {
     }, styles);
     function build(_styles, _empty, key) {
-      const builder = function() {
+      const builder = function () {
         return applyStyle.apply(builder, arguments);
       };
       builder._styles = _styles;
@@ -4127,7 +4127,7 @@ var require_lib2 = __commonJS({
     function _getRequireWildcardCache(e) {
       if ("function" != typeof WeakMap) return null;
       var r = /* @__PURE__ */ new WeakMap(), t = /* @__PURE__ */ new WeakMap();
-      return (_getRequireWildcardCache = function(e2) {
+      return (_getRequireWildcardCache = function (e2) {
         return e2 ? t : r;
       })(e);
     }
@@ -4162,7 +4162,7 @@ var require_lib2 = __commonJS({
     var tokenize;
     {
       const JSX_TAG = /^[a-z][\w-]*$/i;
-      const getTokenType = function(token, offset, text) {
+      const getTokenType = function (token, offset, text) {
         if (token.type === "name") {
           if ((0, _helperValidatorIdentifier.isKeyword)(token.value) || (0, _helperValidatorIdentifier.isStrictReservedWord)(token.value, true) || sometimesKeywords.has(token.value)) {
             return "keyword";
@@ -4251,7 +4251,7 @@ var require_lib3 = __commonJS({
     function _getRequireWildcardCache(e) {
       if ("function" != typeof WeakMap) return null;
       var r = /* @__PURE__ */ new WeakMap(), t = /* @__PURE__ */ new WeakMap();
-      return (_getRequireWildcardCache = function(e2) {
+      return (_getRequireWildcardCache = function (e2) {
         return e2 ? t : r;
       })(e);
     }
@@ -4432,7 +4432,7 @@ var require_constants = __commonJS({
     var SEMVER_SPEC_VERSION = "2.0.0";
     var MAX_LENGTH = 256;
     var MAX_SAFE_INTEGER = Number.MAX_SAFE_INTEGER || /* istanbul ignore next */
-    9007199254740991;
+      9007199254740991;
     var MAX_SAFE_COMPONENT_LENGTH = 16;
     var MAX_SAFE_BUILD_LENGTH = MAX_LENGTH - 6;
     var RELEASE_TYPES = [
@@ -5569,7 +5569,7 @@ var require_scan = __commonJS({
     "use strict";
     var licenses = [].concat(require_spdx_license_ids()).concat(require_spdx_license_ids());
     var exceptions = require_spdx_exceptions();
-    module2.exports = function(source) {
+    module2.exports = function (source) {
       var index = 0;
       function hasMore() {
         return index < source.length;
@@ -5671,7 +5671,7 @@ var require_scan = __commonJS({
 var require_parse3 = __commonJS({
   "../../node_modules/.pnpm/spdx-expression-parse@3.0.1/node_modules/spdx-expression-parse/parse.js"(exports2, module2) {
     "use strict";
-    module2.exports = function(tokens) {
+    module2.exports = function (tokens) {
       var index = 0;
       function hasMore() {
         return index < tokens.length;
@@ -5787,7 +5787,7 @@ var require_spdx_expression_parse = __commonJS({
     "use strict";
     var scan = require_scan();
     var parse = require_parse3();
-    module2.exports = function(source) {
+    module2.exports = function (source) {
       return parse(scan(source));
     };
   }
@@ -5838,59 +5838,59 @@ var require_spdx_correct = __commonJS({
     var CORRECT = 1;
     var transforms = [
       // e.g. 'mit'
-      function(argument) {
+      function (argument) {
         return argument.toUpperCase();
       },
       // e.g. 'MIT '
-      function(argument) {
+      function (argument) {
         return argument.trim();
       },
       // e.g. 'M.I.T.'
-      function(argument) {
+      function (argument) {
         return argument.replace(/\./g, "");
       },
       // e.g. 'Apache- 2.0'
-      function(argument) {
+      function (argument) {
         return argument.replace(/\s+/g, "");
       },
       // e.g. 'CC BY 4.0''
-      function(argument) {
+      function (argument) {
         return argument.replace(/\s+/g, "-");
       },
       // e.g. 'LGPLv2.1'
-      function(argument) {
+      function (argument) {
         return argument.replace("v", "-");
       },
       // e.g. 'Apache 2.0'
-      function(argument) {
+      function (argument) {
         return argument.replace(/,?\s*(\d)/, "-$1");
       },
       // e.g. 'GPL 2'
-      function(argument) {
+      function (argument) {
         return argument.replace(/,?\s*(\d)/, "-$1.0");
       },
       // e.g. 'Apache Version 2.0'
-      function(argument) {
+      function (argument) {
         return argument.replace(/,?\s*(V\.|v\.|V|v|Version|version)\s*(\d)/, "-$2");
       },
       // e.g. 'Apache Version 2'
-      function(argument) {
+      function (argument) {
         return argument.replace(/,?\s*(V\.|v\.|V|v|Version|version)\s*(\d)/, "-$2.0");
       },
       // e.g. 'ZLIB'
-      function(argument) {
+      function (argument) {
         return argument[0].toUpperCase() + argument.slice(1);
       },
       // e.g. 'MPL/2.0'
-      function(argument) {
+      function (argument) {
         return argument.replace("/", "-");
       },
       // e.g. 'Apache 2'
-      function(argument) {
+      function (argument) {
         return argument.replace(/\s*V\s*(\d)/, "-$1").replace(/(\d)$/, "$1.0");
       },
       // e.g. 'GPL-2.0', 'GPL-3.0'
-      function(argument) {
+      function (argument) {
         if (argument.indexOf("3.0") !== -1) {
           return argument + "-or-later";
         } else {
@@ -5898,62 +5898,62 @@ var require_spdx_correct = __commonJS({
         }
       },
       // e.g. 'GPL-2.0-'
-      function(argument) {
+      function (argument) {
         return argument + "only";
       },
       // e.g. 'GPL2'
-      function(argument) {
+      function (argument) {
         return argument.replace(/(\d)$/, "-$1.0");
       },
       // e.g. 'BSD 3'
-      function(argument) {
+      function (argument) {
         return argument.replace(/(-| )?(\d)$/, "-$2-Clause");
       },
       // e.g. 'BSD clause 3'
-      function(argument) {
+      function (argument) {
         return argument.replace(/(-| )clause(-| )(\d)/, "-$3-Clause");
       },
       // e.g. 'New BSD license'
-      function(argument) {
+      function (argument) {
         return argument.replace(/\b(Modified|New|Revised)(-| )?BSD((-| )License)?/i, "BSD-3-Clause");
       },
       // e.g. 'Simplified BSD license'
-      function(argument) {
+      function (argument) {
         return argument.replace(/\bSimplified(-| )?BSD((-| )License)?/i, "BSD-2-Clause");
       },
       // e.g. 'Free BSD license'
-      function(argument) {
+      function (argument) {
         return argument.replace(/\b(Free|Net)(-| )?BSD((-| )License)?/i, "BSD-2-Clause-$1BSD");
       },
       // e.g. 'Clear BSD license'
-      function(argument) {
+      function (argument) {
         return argument.replace(/\bClear(-| )?BSD((-| )License)?/i, "BSD-3-Clause-Clear");
       },
       // e.g. 'Old BSD License'
-      function(argument) {
+      function (argument) {
         return argument.replace(/\b(Old|Original)(-| )?BSD((-| )License)?/i, "BSD-4-Clause");
       },
       // e.g. 'BY-NC-4.0'
-      function(argument) {
+      function (argument) {
         return "CC-" + argument;
       },
       // e.g. 'BY-NC'
-      function(argument) {
+      function (argument) {
         return "CC-" + argument + "-4.0";
       },
       // e.g. 'Attribution-NonCommercial'
-      function(argument) {
+      function (argument) {
         return argument.replace("Attribution", "BY").replace("NonCommercial", "NC").replace("NoDerivatives", "ND").replace(/ (\d)/, "-$1").replace(/ ?International/, "");
       },
       // e.g. 'Attribution-NonCommercial'
-      function(argument) {
+      function (argument) {
         return "CC-" + argument.replace("Attribution", "BY").replace("NonCommercial", "NC").replace("NoDerivatives", "ND").replace(/ (\d)/, "-$1").replace(/ ?International/, "") + "-4.0";
       }
     ];
-    var licensesWithVersions = spdxLicenseIds.map(function(id) {
+    var licensesWithVersions = spdxLicenseIds.map(function (id) {
       var match = /^(.*)-\d+\.\d+$/.exec(id);
       return match ? [match[0], match[1]] : [id, null];
-    }).reduce(function(objectMap, item) {
+    }).reduce(function (objectMap, item) {
       var key = item[1];
       objectMap[key] = objectMap[key] || [];
       objectMap[key].push(item[0]);
@@ -6004,7 +6004,7 @@ var require_spdx_correct = __commonJS({
     ].concat(licensesWithOneVersion);
     var SUBSTRING = 0;
     var IDENTIFIER = 1;
-    var validTransformation = function(identifier) {
+    var validTransformation = function (identifier) {
       for (var i = 0; i < transforms.length; i++) {
         var transformed = transforms[i](identifier).trim();
         if (transformed !== identifier && valid(transformed)) {
@@ -6013,7 +6013,7 @@ var require_spdx_correct = __commonJS({
       }
       return null;
     };
-    var validLastResort = function(identifier) {
+    var validLastResort = function (identifier) {
       var upperCased = identifier.toUpperCase();
       for (var i = 0; i < lastResorts.length; i++) {
         var lastResort = lastResorts[i];
@@ -6023,7 +6023,7 @@ var require_spdx_correct = __commonJS({
       }
       return null;
     };
-    var anyCorrection = function(identifier, check) {
+    var anyCorrection = function (identifier, check) {
       for (var i = 0; i < transpositions.length; i++) {
         var transposition = transpositions[i];
         var transposed = transposition[TRANSPOSED];
@@ -6040,7 +6040,7 @@ var require_spdx_correct = __commonJS({
       }
       return null;
     };
-    module2.exports = function(identifier, options) {
+    module2.exports = function (identifier, options) {
       options = options || {};
       var upgrade = options.upgrade === void 0 ? true : !!options.upgrade;
       function postprocess(value) {
@@ -6062,7 +6062,7 @@ var require_spdx_correct = __commonJS({
       if (transformed !== null) {
         return postprocess(transformed);
       }
-      transformed = anyCorrection(identifier, function(argument) {
+      transformed = anyCorrection(identifier, function (argument) {
         if (valid(argument)) {
           return argument;
         }
@@ -6131,7 +6131,7 @@ var require_validate_npm_package_license = __commonJS({
         return usesLicenseRef(ast.left) || usesLicenseRef(ast.right);
       }
     }
-    module2.exports = function(argument) {
+    module2.exports = function (argument) {
       var ast;
       try {
         ast = parse(argument);
@@ -7176,7 +7176,7 @@ var require_cjs = __commonJS({
     _hasFetchMethod = new WeakMap();
     _hasDisposeAfter = new WeakMap();
     _LRUCache_instances = new WeakSet();
-    initializeTTLTracking_fn = function() {
+    initializeTTLTracking_fn = function () {
       const ttls = new ZeroArray(__privateGet(this, _max));
       const starts = new ZeroArray(__privateGet(this, _max));
       __privateSet(this, _ttls, ttls);
@@ -7242,7 +7242,7 @@ var require_cjs = __commonJS({
     _statusTTL = new WeakMap();
     _setItemTTL = new WeakMap();
     _isStale = new WeakMap();
-    initializeSizeTracking_fn = function() {
+    initializeSizeTracking_fn = function () {
       const sizes = new ZeroArray(__privateGet(this, _max));
       __privateSet(this, _calculatedSize, 0);
       __privateSet(this, _sizes, sizes);
@@ -7289,7 +7289,7 @@ var require_cjs = __commonJS({
     _requireSize = new WeakMap();
     indexes_fn = function* ({ allowStale = this.allowStale } = {}) {
       if (__privateGet(this, _size)) {
-        for (let i = __privateGet(this, _tail); true; ) {
+        for (let i = __privateGet(this, _tail); true;) {
           if (!__privateMethod(this, _LRUCache_instances, isValidIndex_fn).call(this, i)) {
             break;
           }
@@ -7306,7 +7306,7 @@ var require_cjs = __commonJS({
     };
     rindexes_fn = function* ({ allowStale = this.allowStale } = {}) {
       if (__privateGet(this, _size)) {
-        for (let i = __privateGet(this, _head); true; ) {
+        for (let i = __privateGet(this, _head); true;) {
           if (!__privateMethod(this, _LRUCache_instances, isValidIndex_fn).call(this, i)) {
             break;
           }
@@ -7321,10 +7321,10 @@ var require_cjs = __commonJS({
         }
       }
     };
-    isValidIndex_fn = function(index) {
+    isValidIndex_fn = function (index) {
       return index !== void 0 && __privateGet(this, _keyMap).get(__privateGet(this, _keyList)[index]) === index;
     };
-    evict_fn = function(free) {
+    evict_fn = function (free) {
       var _a;
       const head = __privateGet(this, _head);
       const k = __privateGet(this, _keyList)[head];
@@ -7355,7 +7355,7 @@ var require_cjs = __commonJS({
       __privateWrapper(this, _size)._--;
       return head;
     };
-    backgroundFetch_fn = function(k, index, options, context) {
+    backgroundFetch_fn = function (k, index, options, context) {
       const v = index === void 0 ? void 0 : __privateGet(this, _valList)[index];
       if (__privateMethod(this, _LRUCache_instances, isBackgroundFetch_fn).call(this, v)) {
         return v;
@@ -7463,17 +7463,17 @@ var require_cjs = __commonJS({
       }
       return bf;
     };
-    isBackgroundFetch_fn = function(p) {
+    isBackgroundFetch_fn = function (p) {
       if (!__privateGet(this, _hasFetchMethod))
         return false;
       const b = p;
       return !!b && b instanceof Promise && b.hasOwnProperty("__staleWhileFetching") && b.__abortController instanceof AC;
     };
-    connect_fn = function(p, n) {
+    connect_fn = function (p, n) {
       __privateGet(this, _prev)[n] = p;
       __privateGet(this, _next)[p] = n;
     };
-    moveToTail_fn = function(index) {
+    moveToTail_fn = function (index) {
       if (index !== __privateGet(this, _tail)) {
         if (index === __privateGet(this, _head)) {
           __privateSet(this, _head, __privateGet(this, _next)[index]);
@@ -7622,7 +7622,7 @@ var require_hosts = __commonJS({
         }
         return { user, project, committish: url.hash.slice(1) };
       },
-      hashformat: function(fragment) {
+      hashformat: function (fragment) {
         return fragment && "file-" + formatHashFragment(fragment);
       }
     };
@@ -7918,7 +7918,7 @@ var require_lib4 = __commonJS({
     _gitHosts = new WeakMap();
     _protocols = new WeakMap();
     _GitHost_instances = new WeakSet();
-    fill_fn = function(template, opts) {
+    fill_fn = function (template, opts) {
       if (typeof template !== "function") {
         return null;
       }
@@ -7978,7 +7978,7 @@ var require_implementation = __commonJS({
       }
       return arr;
     };
-    var joiny = function(arr, joiner) {
+    var joiny = function (arr, joiner) {
       var str = "";
       for (var i = 0; i < arr.length; i += 1) {
         str += arr[i];
@@ -7995,7 +7995,7 @@ var require_implementation = __commonJS({
       }
       var args = slicy(arguments, 1);
       var bound;
-      var binder = function() {
+      var binder = function () {
         if (this instanceof bound) {
           var result = target.apply(
             this,
@@ -8350,9 +8350,9 @@ var require_fixer = __commonJS({
     var isEmail = (str) => str.includes("@") && str.indexOf("@") < str.lastIndexOf(".");
     module2.exports = {
       // default warning function
-      warn: function() {
+      warn: function () {
       },
-      fixRepositoryField: function(data) {
+      fixRepositoryField: function (data) {
         if (data.repositories) {
           this.warn("repositories");
           data.repository = data.repositories[0];
@@ -8377,14 +8377,14 @@ var require_fixer = __commonJS({
           this.warn("brokenGitUrl", r);
         }
       },
-      fixTypos: function(data) {
-        Object.keys(typos.topLevel).forEach(function(d) {
+      fixTypos: function (data) {
+        Object.keys(typos.topLevel).forEach(function (d) {
           if (Object.prototype.hasOwnProperty.call(data, d)) {
             this.warn("typo", d, typos.topLevel[d]);
           }
         }, this);
       },
-      fixScriptsField: function(data) {
+      fixScriptsField: function (data) {
         if (!data.scripts) {
           return;
         }
@@ -8393,7 +8393,7 @@ var require_fixer = __commonJS({
           delete data.scripts;
           return;
         }
-        Object.keys(data.scripts).forEach(function(k) {
+        Object.keys(data.scripts).forEach(function (k) {
           if (typeof data.scripts[k] !== "string") {
             this.warn("nonStringScript");
             delete data.scripts[k];
@@ -8402,13 +8402,13 @@ var require_fixer = __commonJS({
           }
         }, this);
       },
-      fixFilesField: function(data) {
+      fixFilesField: function (data) {
         var files = data.files;
         if (files && !Array.isArray(files)) {
           this.warn("nonArrayFiles");
           delete data.files;
         } else if (data.files) {
-          data.files = data.files.filter(function(file) {
+          data.files = data.files.filter(function (file) {
             if (!file || typeof file !== "string") {
               this.warn("invalidFilename", file);
               return false;
@@ -8418,7 +8418,7 @@ var require_fixer = __commonJS({
           }, this);
         }
       },
-      fixBinField: function(data) {
+      fixBinField: function (data) {
         if (!data.bin) {
           return;
         }
@@ -8433,7 +8433,7 @@ var require_fixer = __commonJS({
           data.bin = b;
         }
       },
-      fixManField: function(data) {
+      fixManField: function (data) {
         if (!data.man) {
           return;
         }
@@ -8441,7 +8441,7 @@ var require_fixer = __commonJS({
           data.man = [data.man];
         }
       },
-      fixBundleDependenciesField: function(data) {
+      fixBundleDependenciesField: function (data) {
         var bdd = "bundledDependencies";
         var bd = "bundleDependencies";
         if (data[bdd] && !data[bd]) {
@@ -8452,7 +8452,7 @@ var require_fixer = __commonJS({
           this.warn("nonArrayBundleDependencies");
           delete data[bd];
         } else if (data[bd]) {
-          data[bd] = data[bd].filter(function(filtered) {
+          data[bd] = data[bd].filter(function (filtered) {
             if (!filtered || typeof filtered !== "string") {
               this.warn("nonStringBundleDependency", filtered);
               return false;
@@ -8469,11 +8469,11 @@ var require_fixer = __commonJS({
           }, this);
         }
       },
-      fixDependencies: function(data, strict) {
+      fixDependencies: function (data, strict) {
         objectifyDeps(data, this.warn);
         addOptionalDepsToDeps(data, this.warn);
         this.fixBundleDependenciesField(data);
-        ["dependencies", "devDependencies"].forEach(function(deps) {
+        ["dependencies", "devDependencies"].forEach(function (deps) {
           if (!(deps in data)) {
             return;
           }
@@ -8482,7 +8482,7 @@ var require_fixer = __commonJS({
             delete data[deps];
             return;
           }
-          Object.keys(data[deps]).forEach(function(d) {
+          Object.keys(data[deps]).forEach(function (d) {
             var r = data[deps][d];
             if (typeof r !== "string") {
               this.warn("nonStringDependency", d, JSON.stringify(r));
@@ -8495,13 +8495,13 @@ var require_fixer = __commonJS({
           }, this);
         }, this);
       },
-      fixModulesField: function(data) {
+      fixModulesField: function (data) {
         if (data.modules) {
           this.warn("deprecatedModules");
           delete data.modules;
         }
       },
-      fixKeywordsField: function(data) {
+      fixKeywordsField: function (data) {
         if (typeof data.keywords === "string") {
           data.keywords = data.keywords.split(/,\s+/);
         }
@@ -8509,7 +8509,7 @@ var require_fixer = __commonJS({
           delete data.keywords;
           this.warn("nonArrayKeywords");
         } else if (data.keywords) {
-          data.keywords = data.keywords.filter(function(kw) {
+          data.keywords = data.keywords.filter(function (kw) {
             if (typeof kw !== "string" || !kw) {
               this.warn("nonStringKeyword");
               return false;
@@ -8519,7 +8519,7 @@ var require_fixer = __commonJS({
           }, this);
         }
       },
-      fixVersionField: function(data, strict) {
+      fixVersionField: function (data, strict) {
         var loose = !strict;
         if (!data.version) {
           data.version = "";
@@ -8531,11 +8531,11 @@ var require_fixer = __commonJS({
         data.version = cleanSemver(data.version, loose);
         return true;
       },
-      fixPeople: function(data) {
+      fixPeople: function (data) {
         modifyPeople(data, unParsePerson);
         modifyPeople(data, parsePerson);
       },
-      fixNameField: function(data, options) {
+      fixNameField: function (data, options) {
         if (typeof options === "boolean") {
           options = { strict: options };
         } else if (typeof options === "undefined") {
@@ -8557,7 +8557,7 @@ var require_fixer = __commonJS({
           this.warn("conflictingName", data.name);
         }
       },
-      fixDescriptionField: function(data) {
+      fixDescriptionField: function (data) {
         if (data.description && typeof data.description !== "string") {
           this.warn("nonStringDescription");
           delete data.description;
@@ -8572,13 +8572,13 @@ var require_fixer = __commonJS({
           this.warn("missingDescription");
         }
       },
-      fixReadmeField: function(data) {
+      fixReadmeField: function (data) {
         if (!data.readme) {
           this.warn("missingReadme");
           data.readme = "ERROR: No README data found!";
         }
       },
-      fixBugsField: function(data) {
+      fixBugsField: function (data) {
         if (!data.bugs && data.repository && data.repository.url) {
           var hosted = hostedGitInfo.fromUrl(data.repository.url);
           if (hosted && hosted.bugs()) {
@@ -8618,7 +8618,7 @@ var require_fixer = __commonJS({
           }
         }
       },
-      fixHomepageField: function(data) {
+      fixHomepageField: function (data) {
         if (!data.homepage && data.repository && data.repository.url) {
           var hosted = hostedGitInfo.fromUrl(data.repository.url);
           if (hosted && hosted.docs()) {
@@ -8636,7 +8636,7 @@ var require_fixer = __commonJS({
           data.homepage = "http://" + data.homepage;
         }
       },
-      fixLicenseField: function(data) {
+      fixLicenseField: function (data) {
         const license = data.license || data.licence;
         if (!license) {
           return this.warn("missingLicense");
@@ -8671,7 +8671,7 @@ var require_fixer = __commonJS({
       if (data.author) {
         data.author = fn(data.author);
       }
-      ["maintainers", "contributors"].forEach(function(set) {
+      ["maintainers", "contributors"].forEach(function (set) {
         if (!Array.isArray(data[set])) {
           return;
         }
@@ -8715,7 +8715,7 @@ var require_fixer = __commonJS({
         return;
       }
       var d = data.dependencies || {};
-      Object.keys(o).forEach(function(k) {
+      Object.keys(o).forEach(function (k) {
         d[k] = o[k];
       });
       data.dependencies = d;
@@ -8732,9 +8732,9 @@ var require_fixer = __commonJS({
       }
       warn("deprecatedArrayDependencies", type);
       var o = {};
-      deps.filter(function(d) {
+      deps.filter(function (d) {
         return typeof d === "string";
-      }).forEach(function(d) {
+      }).forEach(function (d) {
         d = d.trim().split(/(:?[@\s><=])/);
         var dn = d.shift();
         var dv = d.join("");
@@ -8745,7 +8745,7 @@ var require_fixer = __commonJS({
       return o;
     }
     function objectifyDeps(data, warn) {
-      depTypes.forEach(function(type) {
+      depTypes.forEach(function (type) {
         if (!data[type]) {
           return;
         }
@@ -8756,7 +8756,7 @@ var require_fixer = __commonJS({
       if (!bugs) {
         return;
       }
-      Object.keys(bugs).forEach(function(k) {
+      Object.keys(bugs).forEach(function (k) {
         if (typos.bugs[k]) {
           warn("typo", k, typos.bugs[k], "bugs");
           bugs[typos.bugs[k]] = bugs[k];
@@ -8809,7 +8809,7 @@ var require_make_warning = __commonJS({
     "use strict";
     var util2 = require("util");
     var messages = require_warning_messages();
-    module2.exports = function() {
+    module2.exports = function () {
       var args = Array.prototype.slice.call(arguments, 0);
       var warningName = args.shift();
       if (warningName === "typo") {
@@ -8855,7 +8855,7 @@ var require_normalize = __commonJS({
       "license"
     ];
     var otherThingsToFix = ["dependencies", "people", "typos"];
-    var thingsToFix = fieldsToFix.map(function(fieldName) {
+    var thingsToFix = fieldsToFix.map(function (fieldName) {
       return ucFirst(fieldName) + "Field";
     });
     thingsToFix = thingsToFix.concat(otherThingsToFix);
@@ -8868,16 +8868,16 @@ var require_normalize = __commonJS({
         strict = false;
       }
       if (!warn || data.private) {
-        warn = function(msg) {
+        warn = function (msg) {
         };
       }
       if (data.scripts && data.scripts.install === "node-gyp rebuild" && !data.scripts.preinstall) {
         data.gypfile = true;
       }
-      fixer.warn = function() {
+      fixer.warn = function () {
         warn(makeWarning.apply(null, arguments));
       };
-      thingsToFix.forEach(function(thingName) {
+      thingsToFix.forEach(function (thingName) {
         fixer["fix" + ucFirst(thingName)](data, strict);
       });
       data._id = data.name + "@" + data.version;
@@ -8892,8 +8892,8 @@ var require_normalize = __commonJS({
 var require_universalify = __commonJS({
   "../../node_modules/.pnpm/universalify@2.0.0/node_modules/universalify/index.js"(exports2) {
     "use strict";
-    exports2.fromCallback = function(fn) {
-      return Object.defineProperty(function(...args) {
+    exports2.fromCallback = function (fn) {
+      return Object.defineProperty(function (...args) {
         if (typeof args[args.length - 1] === "function") fn.apply(this, args);
         else {
           return new Promise((resolve, reject) => {
@@ -8906,8 +8906,8 @@ var require_universalify = __commonJS({
         }
       }, "name", { value: fn.name });
     };
-    exports2.fromPromise = function(fn) {
-      return Object.defineProperty(function(...args) {
+    exports2.fromPromise = function (fn) {
+      return Object.defineProperty(function (...args) {
         const cb = args[args.length - 1];
         if (typeof cb !== "function") return fn.apply(this, args);
         else fn.apply(this, args.slice(0, -1)).then((r) => cb(null, r), cb);
@@ -8924,7 +8924,7 @@ var require_polyfills = __commonJS({
     var origCwd = process.cwd;
     var cwd2 = null;
     var platform = process.env.GRACEFUL_FS_PLATFORM || process.platform;
-    process.cwd = function() {
+    process.cwd = function () {
       if (!cwd2)
         cwd2 = origCwd.call(process);
       return cwd2;
@@ -8935,7 +8935,7 @@ var require_polyfills = __commonJS({
     }
     if (typeof process.chdir === "function") {
       chdir = process.chdir;
-      process.chdir = function(d) {
+      process.chdir = function (d) {
         cwd2 = null;
         chdir.call(process, d);
       };
@@ -8969,28 +8969,28 @@ var require_polyfills = __commonJS({
       fs8.fstatSync = statFixSync(fs8.fstatSync);
       fs8.lstatSync = statFixSync(fs8.lstatSync);
       if (fs8.chmod && !fs8.lchmod) {
-        fs8.lchmod = function(path10, mode, cb) {
+        fs8.lchmod = function (path10, mode, cb) {
           if (cb) process.nextTick(cb);
         };
-        fs8.lchmodSync = function() {
+        fs8.lchmodSync = function () {
         };
       }
       if (fs8.chown && !fs8.lchown) {
-        fs8.lchown = function(path10, uid, gid, cb) {
+        fs8.lchown = function (path10, uid, gid, cb) {
           if (cb) process.nextTick(cb);
         };
-        fs8.lchownSync = function() {
+        fs8.lchownSync = function () {
         };
       }
       if (platform === "win32") {
-        fs8.rename = typeof fs8.rename !== "function" ? fs8.rename : function(fs$rename) {
+        fs8.rename = typeof fs8.rename !== "function" ? fs8.rename : function (fs$rename) {
           function rename(from, to, cb) {
             var start = Date.now();
             var backoff = 0;
             fs$rename(from, to, function CB(er) {
               if (er && (er.code === "EACCES" || er.code === "EPERM") && Date.now() - start < 6e4) {
-                setTimeout(function() {
-                  fs8.stat(to, function(stater, st) {
+                setTimeout(function () {
+                  fs8.stat(to, function (stater, st) {
                     if (stater && stater.code === "ENOENT")
                       fs$rename(from, to, CB);
                     else
@@ -9008,12 +9008,12 @@ var require_polyfills = __commonJS({
           return rename;
         }(fs8.rename);
       }
-      fs8.read = typeof fs8.read !== "function" ? fs8.read : function(fs$read) {
+      fs8.read = typeof fs8.read !== "function" ? fs8.read : function (fs$read) {
         function read(fd, buffer, offset, length, position, callback_) {
           var callback;
           if (callback_ && typeof callback_ === "function") {
             var eagCounter = 0;
-            callback = function(er, _, __) {
+            callback = function (er, _, __) {
               if (er && er.code === "EAGAIN" && eagCounter < 10) {
                 eagCounter++;
                 return fs$read.call(fs8, fd, buffer, offset, length, position, callback);
@@ -9026,8 +9026,8 @@ var require_polyfills = __commonJS({
         if (Object.setPrototypeOf) Object.setPrototypeOf(read, fs$read);
         return read;
       }(fs8.read);
-      fs8.readSync = typeof fs8.readSync !== "function" ? fs8.readSync : /* @__PURE__ */ function(fs$readSync) {
-        return function(fd, buffer, offset, length, position) {
+      fs8.readSync = typeof fs8.readSync !== "function" ? fs8.readSync : /* @__PURE__ */ function (fs$readSync) {
+        return function (fd, buffer, offset, length, position) {
           var eagCounter = 0;
           while (true) {
             try {
@@ -9043,25 +9043,25 @@ var require_polyfills = __commonJS({
         };
       }(fs8.readSync);
       function patchLchmod(fs9) {
-        fs9.lchmod = function(path10, mode, callback) {
+        fs9.lchmod = function (path10, mode, callback) {
           fs9.open(
             path10,
             constants.O_WRONLY | constants.O_SYMLINK,
             mode,
-            function(err, fd) {
+            function (err, fd) {
               if (err) {
                 if (callback) callback(err);
                 return;
               }
-              fs9.fchmod(fd, mode, function(err2) {
-                fs9.close(fd, function(err22) {
+              fs9.fchmod(fd, mode, function (err2) {
+                fs9.close(fd, function (err22) {
                   if (callback) callback(err2 || err22);
                 });
               });
             }
           );
         };
-        fs9.lchmodSync = function(path10, mode) {
+        fs9.lchmodSync = function (path10, mode) {
           var fd = fs9.openSync(path10, constants.O_WRONLY | constants.O_SYMLINK, mode);
           var threw = true;
           var ret;
@@ -9083,20 +9083,20 @@ var require_polyfills = __commonJS({
       }
       function patchLutimes(fs9) {
         if (constants.hasOwnProperty("O_SYMLINK") && fs9.futimes) {
-          fs9.lutimes = function(path10, at, mt, cb) {
-            fs9.open(path10, constants.O_SYMLINK, function(er, fd) {
+          fs9.lutimes = function (path10, at, mt, cb) {
+            fs9.open(path10, constants.O_SYMLINK, function (er, fd) {
               if (er) {
                 if (cb) cb(er);
                 return;
               }
-              fs9.futimes(fd, at, mt, function(er2) {
-                fs9.close(fd, function(er22) {
+              fs9.futimes(fd, at, mt, function (er2) {
+                fs9.close(fd, function (er22) {
                   if (cb) cb(er2 || er22);
                 });
               });
             });
           };
-          fs9.lutimesSync = function(path10, at, mt) {
+          fs9.lutimesSync = function (path10, at, mt) {
             var fd = fs9.openSync(path10, constants.O_SYMLINK);
             var ret;
             var threw = true;
@@ -9116,17 +9116,17 @@ var require_polyfills = __commonJS({
             return ret;
           };
         } else if (fs9.futimes) {
-          fs9.lutimes = function(_a, _b, _c, cb) {
+          fs9.lutimes = function (_a, _b, _c, cb) {
             if (cb) process.nextTick(cb);
           };
-          fs9.lutimesSync = function() {
+          fs9.lutimesSync = function () {
           };
         }
       }
       function chmodFix(orig) {
         if (!orig) return orig;
-        return function(target, mode, cb) {
-          return orig.call(fs8, target, mode, function(er) {
+        return function (target, mode, cb) {
+          return orig.call(fs8, target, mode, function (er) {
             if (chownErOk(er)) er = null;
             if (cb) cb.apply(this, arguments);
           });
@@ -9134,7 +9134,7 @@ var require_polyfills = __commonJS({
       }
       function chmodFixSync(orig) {
         if (!orig) return orig;
-        return function(target, mode) {
+        return function (target, mode) {
           try {
             return orig.call(fs8, target, mode);
           } catch (er) {
@@ -9144,8 +9144,8 @@ var require_polyfills = __commonJS({
       }
       function chownFix(orig) {
         if (!orig) return orig;
-        return function(target, uid, gid, cb) {
-          return orig.call(fs8, target, uid, gid, function(er) {
+        return function (target, uid, gid, cb) {
+          return orig.call(fs8, target, uid, gid, function (er) {
             if (chownErOk(er)) er = null;
             if (cb) cb.apply(this, arguments);
           });
@@ -9153,7 +9153,7 @@ var require_polyfills = __commonJS({
       }
       function chownFixSync(orig) {
         if (!orig) return orig;
-        return function(target, uid, gid) {
+        return function (target, uid, gid) {
           try {
             return orig.call(fs8, target, uid, gid);
           } catch (er) {
@@ -9163,7 +9163,7 @@ var require_polyfills = __commonJS({
       }
       function statFix(orig) {
         if (!orig) return orig;
-        return function(target, options, cb) {
+        return function (target, options, cb) {
           if (typeof options === "function") {
             cb = options;
             options = null;
@@ -9180,7 +9180,7 @@ var require_polyfills = __commonJS({
       }
       function statFixSync(orig) {
         if (!orig) return orig;
-        return function(target, options) {
+        return function (target, options) {
           var stats = options ? orig.call(fs8, target, options) : orig.call(fs8, target);
           if (stats) {
             if (stats.uid < 0) stats.uid += 4294967296;
@@ -9249,12 +9249,12 @@ var require_legacy_streams = __commonJS({
           this.pos = this.start;
         }
         if (this.fd !== null) {
-          process.nextTick(function() {
+          process.nextTick(function () {
             self._read();
           });
           return;
         }
-        fs8.open(this.path, this.flags, this.mode, function(err, fd) {
+        fs8.open(this.path, this.flags, this.mode, function (err, fd) {
           if (err) {
             self.emit("error", err);
             self.readable = false;
@@ -9307,7 +9307,7 @@ var require_clone = __commonJS({
   "../../node_modules/.pnpm/graceful-fs@4.2.10/node_modules/graceful-fs/clone.js"(exports2, module2) {
     "use strict";
     module2.exports = clone;
-    var getPrototypeOf = Object.getPrototypeOf || function(obj) {
+    var getPrototypeOf = Object.getPrototypeOf || function (obj) {
       return obj.__proto__;
     };
     function clone(obj) {
@@ -9317,7 +9317,7 @@ var require_clone = __commonJS({
         var copy = { __proto__: getPrototypeOf(obj) };
       else
         var copy = /* @__PURE__ */ Object.create(null);
-      Object.getOwnPropertyNames(obj).forEach(function(key) {
+      Object.getOwnPropertyNames(obj).forEach(function (key) {
         Object.defineProperty(copy, key, Object.getOwnPropertyDescriptor(obj, key));
       });
       return copy;
@@ -9347,7 +9347,7 @@ var require_graceful_fs = __commonJS({
     }
     function publishQueue(context, queue2) {
       Object.defineProperty(context, gracefulQueue, {
-        get: function() {
+        get: function () {
           return queue2;
         }
       });
@@ -9356,7 +9356,7 @@ var require_graceful_fs = __commonJS({
     if (util2.debuglog)
       debug7 = util2.debuglog("gfs4");
     else if (/\bgfs4\b/i.test(process.env.NODE_DEBUG || ""))
-      debug7 = function() {
+      debug7 = function () {
         var m = util2.format.apply(util2, arguments);
         m = "GFS4: " + m.split(/\n/).join("\nGFS4: ");
         console.error(m);
@@ -9364,9 +9364,9 @@ var require_graceful_fs = __commonJS({
     if (!fs8[gracefulQueue]) {
       queue = global[gracefulQueue] || [];
       publishQueue(fs8, queue);
-      fs8.close = function(fs$close) {
+      fs8.close = function (fs$close) {
         function close(fd, cb) {
-          return fs$close.call(fs8, fd, function(err) {
+          return fs$close.call(fs8, fd, function (err) {
             if (!err) {
               resetQueue();
             }
@@ -9379,7 +9379,7 @@ var require_graceful_fs = __commonJS({
         });
         return close;
       }(fs8.close);
-      fs8.closeSync = function(fs$closeSync) {
+      fs8.closeSync = function (fs$closeSync) {
         function closeSync(fd) {
           fs$closeSync.apply(fs8, arguments);
           resetQueue();
@@ -9390,7 +9390,7 @@ var require_graceful_fs = __commonJS({
         return closeSync;
       }(fs8.closeSync);
       if (/\bgfs4\b/i.test(process.env.NODE_DEBUG || "")) {
-        process.on("exit", function() {
+        process.on("exit", function () {
           debug7(fs8[gracefulQueue]);
           require("assert").equal(fs8[gracefulQueue].length, 0);
         });
@@ -9417,7 +9417,7 @@ var require_graceful_fs = __commonJS({
           cb = options, options = null;
         return go$readFile(path10, options, cb);
         function go$readFile(path11, options2, cb2, startTime) {
-          return fs$readFile(path11, options2, function(err) {
+          return fs$readFile(path11, options2, function (err) {
             if (err && (err.code === "EMFILE" || err.code === "ENFILE"))
               enqueue([go$readFile, [path11, options2, cb2], err, startTime || Date.now(), Date.now()]);
             else {
@@ -9434,7 +9434,7 @@ var require_graceful_fs = __commonJS({
           cb = options, options = null;
         return go$writeFile(path10, data, options, cb);
         function go$writeFile(path11, data2, options2, cb2, startTime) {
-          return fs$writeFile(path11, data2, options2, function(err) {
+          return fs$writeFile(path11, data2, options2, function (err) {
             if (err && (err.code === "EMFILE" || err.code === "ENFILE"))
               enqueue([go$writeFile, [path11, data2, options2, cb2], err, startTime || Date.now(), Date.now()]);
             else {
@@ -9452,7 +9452,7 @@ var require_graceful_fs = __commonJS({
           cb = options, options = null;
         return go$appendFile(path10, data, options, cb);
         function go$appendFile(path11, data2, options2, cb2, startTime) {
-          return fs$appendFile(path11, data2, options2, function(err) {
+          return fs$appendFile(path11, data2, options2, function (err) {
             if (err && (err.code === "EMFILE" || err.code === "ENFILE"))
               enqueue([go$appendFile, [path11, data2, options2, cb2], err, startTime || Date.now(), Date.now()]);
             else {
@@ -9472,7 +9472,7 @@ var require_graceful_fs = __commonJS({
         }
         return go$copyFile(src, dest, flags, cb);
         function go$copyFile(src2, dest2, flags2, cb2, startTime) {
-          return fs$copyFile(src2, dest2, flags2, function(err) {
+          return fs$copyFile(src2, dest2, flags2, function (err) {
             if (err && (err.code === "EMFILE" || err.code === "ENFILE"))
               enqueue([go$copyFile, [src2, dest2, flags2, cb2], err, startTime || Date.now(), Date.now()]);
             else {
@@ -9505,7 +9505,7 @@ var require_graceful_fs = __commonJS({
         };
         return go$readdir(path10, options, cb);
         function fs$readdirCallback(path11, options2, cb2, startTime) {
-          return function(err, files) {
+          return function (err, files) {
             if (err && (err.code === "EMFILE" || err.code === "ENFILE"))
               enqueue([
                 go$readdir,
@@ -9539,20 +9539,20 @@ var require_graceful_fs = __commonJS({
         WriteStream.prototype.open = WriteStream$open;
       }
       Object.defineProperty(fs9, "ReadStream", {
-        get: function() {
+        get: function () {
           return ReadStream;
         },
-        set: function(val) {
+        set: function (val) {
           ReadStream = val;
         },
         enumerable: true,
         configurable: true
       });
       Object.defineProperty(fs9, "WriteStream", {
-        get: function() {
+        get: function () {
           return WriteStream;
         },
-        set: function(val) {
+        set: function (val) {
           WriteStream = val;
         },
         enumerable: true,
@@ -9560,10 +9560,10 @@ var require_graceful_fs = __commonJS({
       });
       var FileReadStream = ReadStream;
       Object.defineProperty(fs9, "FileReadStream", {
-        get: function() {
+        get: function () {
           return FileReadStream;
         },
-        set: function(val) {
+        set: function (val) {
           FileReadStream = val;
         },
         enumerable: true,
@@ -9571,10 +9571,10 @@ var require_graceful_fs = __commonJS({
       });
       var FileWriteStream = WriteStream;
       Object.defineProperty(fs9, "FileWriteStream", {
-        get: function() {
+        get: function () {
           return FileWriteStream;
         },
-        set: function(val) {
+        set: function (val) {
           FileWriteStream = val;
         },
         enumerable: true,
@@ -9588,7 +9588,7 @@ var require_graceful_fs = __commonJS({
       }
       function ReadStream$open() {
         var that = this;
-        open(that.path, that.flags, that.mode, function(err, fd) {
+        open(that.path, that.flags, that.mode, function (err, fd) {
           if (err) {
             if (that.autoClose)
               that.destroy();
@@ -9608,7 +9608,7 @@ var require_graceful_fs = __commonJS({
       }
       function WriteStream$open() {
         var that = this;
-        open(that.path, that.flags, that.mode, function(err, fd) {
+        open(that.path, that.flags, that.mode, function (err, fd) {
           if (err) {
             that.destroy();
             that.emit("error", err);
@@ -9631,7 +9631,7 @@ var require_graceful_fs = __commonJS({
           cb = mode, mode = null;
         return go$open(path10, flags, mode, cb);
         function go$open(path11, flags2, mode2, cb2, startTime) {
-          return fs$open(path11, flags2, mode2, function(err, fd) {
+          return fs$open(path11, flags2, mode2, function (err, fd) {
             if (err && (err.code === "EMFILE" || err.code === "ENFILE"))
               enqueue([go$open, [path11, flags2, mode2, cb2], err, startTime || Date.now(), Date.now()]);
             else {
@@ -9744,7 +9744,7 @@ var require_fs = __commonJS({
     api.forEach((method2) => {
       exports2[method2] = u(fs8[method2]);
     });
-    exports2.exists = function(filename, callback) {
+    exports2.exists = function (filename, callback) {
       if (typeof callback === "function") {
         return fs8.exists(filename, callback);
       }
@@ -9752,7 +9752,7 @@ var require_fs = __commonJS({
         return fs8.exists(filename, resolve);
       });
     };
-    exports2.read = function(fd, buffer, offset, length, position, callback) {
+    exports2.read = function (fd, buffer, offset, length, position, callback) {
       if (typeof callback === "function") {
         return fs8.read(fd, buffer, offset, length, position, callback);
       }
@@ -9763,7 +9763,7 @@ var require_fs = __commonJS({
         });
       });
     };
-    exports2.write = function(fd, buffer, ...args) {
+    exports2.write = function (fd, buffer, ...args) {
       if (typeof args[args.length - 1] === "function") {
         return fs8.write(fd, buffer, ...args);
       }
@@ -9774,7 +9774,7 @@ var require_fs = __commonJS({
         });
       });
     };
-    exports2.readv = function(fd, buffers, ...args) {
+    exports2.readv = function (fd, buffers, ...args) {
       if (typeof args[args.length - 1] === "function") {
         return fs8.readv(fd, buffers, ...args);
       }
@@ -9785,7 +9785,7 @@ var require_fs = __commonJS({
         });
       });
     };
-    exports2.writev = function(fd, buffers, ...args) {
+    exports2.writev = function (fd, buffers, ...args) {
       if (typeof args[args.length - 1] === "function") {
         return fs8.writev(fd, buffers, ...args);
       }
@@ -10064,7 +10064,7 @@ var require_copy = __commonJS({
       } else if (typeof opts === "function") {
         opts = { filter: opts };
       }
-      cb = cb || function() {
+      cb = cb || function () {
       };
       opts = opts || {};
       opts.clobber = "clobber" in opts ? !!opts.clobber : true;
@@ -11280,7 +11280,7 @@ var require_main2 = __commonJS({
       }
       try {
         const parsed = DotenvModule.parse(fs8.readFileSync(dotenvPath, { encoding }));
-        Object.keys(parsed).forEach(function(key) {
+        Object.keys(parsed).forEach(function (key) {
           if (!Object.prototype.hasOwnProperty.call(process.env, key)) {
             process.env[key] = parsed[key];
           } else {
@@ -11352,17 +11352,17 @@ var require_indent_string = __commonJS({
 var require_pluralize = __commonJS({
   "../../node_modules/.pnpm/pluralize@8.0.0/node_modules/pluralize/pluralize.js"(exports2, module2) {
     "use strict";
-    (function(root, pluralize3) {
+    (function (root, pluralize3) {
       if (typeof require === "function" && typeof exports2 === "object" && typeof module2 === "object") {
         module2.exports = pluralize3();
       } else if (typeof define === "function" && define.amd) {
-        define(function() {
+        define(function () {
           return pluralize3();
         });
       } else {
         root.pluralize = pluralize3();
       }
-    })(exports2, function() {
+    })(exports2, function () {
       var pluralRules = [];
       var singularRules = [];
       var uncountables = {};
@@ -11384,12 +11384,12 @@ var require_pluralize = __commonJS({
         return token.toLowerCase();
       }
       function interpolate(str, args) {
-        return str.replace(/\$(\d{1,2})/g, function(match, index) {
+        return str.replace(/\$(\d{1,2})/g, function (match, index) {
           return args[index] || "";
         });
       }
       function replace(word, rule) {
-        return word.replace(rule[0], function(match, index) {
+        return word.replace(rule[0], function (match, index) {
           var result = interpolate(rule[1], arguments);
           if (match === "") {
             return restoreCase(word[index - 1], result);
@@ -11409,7 +11409,7 @@ var require_pluralize = __commonJS({
         return word;
       }
       function replaceWord(replaceMap, keepMap, rules) {
-        return function(word) {
+        return function (word) {
           var token = word.toLowerCase();
           if (keepMap.hasOwnProperty(token)) {
             return restoreCase(word, token);
@@ -11421,7 +11421,7 @@ var require_pluralize = __commonJS({
         };
       }
       function checkWord(replaceMap, keepMap, rules, bool) {
-        return function(word) {
+        return function (word) {
           var token = word.toLowerCase();
           if (keepMap.hasOwnProperty(token)) return true;
           if (replaceMap.hasOwnProperty(token)) return false;
@@ -11452,13 +11452,13 @@ var require_pluralize = __commonJS({
         irregularSingles,
         singularRules
       );
-      pluralize3.addPluralRule = function(rule, replacement) {
+      pluralize3.addPluralRule = function (rule, replacement) {
         pluralRules.push([sanitizeRule(rule), replacement]);
       };
-      pluralize3.addSingularRule = function(rule, replacement) {
+      pluralize3.addSingularRule = function (rule, replacement) {
         singularRules.push([sanitizeRule(rule), replacement]);
       };
-      pluralize3.addUncountableRule = function(word) {
+      pluralize3.addUncountableRule = function (word) {
         if (typeof word === "string") {
           uncountables[word.toLowerCase()] = true;
           return;
@@ -11466,7 +11466,7 @@ var require_pluralize = __commonJS({
         pluralize3.addPluralRule(word, "$0");
         pluralize3.addSingularRule(word, "$0");
       };
-      pluralize3.addIrregularRule = function(single, plural) {
+      pluralize3.addIrregularRule = function (single, plural) {
         plural = plural.toLowerCase();
         single = single.toLowerCase();
         irregularSingles[single] = plural;
@@ -11525,7 +11525,7 @@ var require_pluralize = __commonJS({
         ["groove", "grooves"],
         ["pickaxe", "pickaxes"],
         ["passerby", "passersby"]
-      ].forEach(function(rule) {
+      ].forEach(function (rule) {
         return pluralize3.addIrregularRule(rule[0], rule[1]);
       });
       [
@@ -11554,7 +11554,7 @@ var require_pluralize = __commonJS({
         [/eaux$/i, "$0"],
         [/m[ae]n$/i, "men"],
         ["thou", "you"]
-      ].forEach(function(rule) {
+      ].forEach(function (rule) {
         return pluralize3.addPluralRule(rule[0], rule[1]);
       });
       [
@@ -11581,7 +11581,7 @@ var require_pluralize = __commonJS({
         [/(child)ren$/i, "$1"],
         [/(eau)x?$/i, "$1"],
         [/men$/i, "man"]
-      ].forEach(function(rule) {
+      ].forEach(function (rule) {
         return pluralize3.addSingularRule(rule[0], rule[1]);
       });
       [
@@ -12230,9 +12230,9 @@ var require_array_species_create = __commonJS({
     Object.defineProperty(exports2, "__esModule", {
       value: true
     });
-    var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function(obj) {
+    var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) {
       return typeof obj;
-    } : function(obj) {
+    } : function (obj) {
       return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj;
     };
     exports2.default = arraySpeciesCreate;
@@ -12319,7 +12319,7 @@ var require_flatten = __commonJS({
         var a = (0, _arraySpeciesCreate2.default)(o, this.length);
         var depthNum = depth !== void 0 ? Number(depth) : Infinity;
         (0, _flattenIntoArray2.default)(a, o, 0, depthNum);
-        return a.filter(function(e) {
+        return a.filter(function (e) {
           return e !== void 0;
         });
       };
@@ -12356,7 +12356,7 @@ var require_flat_map = __commonJS({
           callbackFn,
           t
         );
-        return a.filter(function(x) {
+        return a.filter(function (x) {
           return x !== void 0;
         }, a);
       };
@@ -12770,15 +12770,15 @@ var require_ci_info = __commonJS({
     var vendors = require_vendors();
     var env2 = process.env;
     Object.defineProperty(exports2, "_vendors", {
-      value: vendors.map(function(v) {
+      value: vendors.map(function (v) {
         return v.constant;
       })
     });
     exports2.name = null;
     exports2.isPR = null;
-    vendors.forEach(function(vendor) {
+    vendors.forEach(function (vendor) {
       const envs = Array.isArray(vendor.env) ? vendor.env : [vendor.env];
-      const isCI = envs.every(function(obj) {
+      const isCI = envs.every(function (obj) {
         return checkEnv(obj);
       });
       exports2[vendor.constant] = isCI;
@@ -12794,7 +12794,7 @@ var require_ci_info = __commonJS({
           if ("env" in vendor.pr) {
             exports2.isPR = vendor.pr.env in env2 && env2[vendor.pr.env] !== vendor.pr.ne;
           } else if ("any" in vendor.pr) {
-            exports2.isPR = vendor.pr.any.some(function(key) {
+            exports2.isPR = vendor.pr.any.some(function (key) {
               return !!env2[key];
             });
           } else {
@@ -12806,27 +12806,27 @@ var require_ci_info = __commonJS({
       }
     });
     exports2.isCI = !!(env2.CI !== "false" && // Bypass all checks if CI env is explicitly set to 'false'
-    (env2.BUILD_ID || // Jenkins, Cloudbees
-    env2.BUILD_NUMBER || // Jenkins, TeamCity
-    env2.CI || // Travis CI, CircleCI, Cirrus CI, Gitlab CI, Appveyor, CodeShip, dsari
-    env2.CI_APP_ID || // Appflow
-    env2.CI_BUILD_ID || // Appflow
-    env2.CI_BUILD_NUMBER || // Appflow
-    env2.CI_NAME || // Codeship and others
-    env2.CONTINUOUS_INTEGRATION || // Travis CI, Cirrus CI
-    env2.RUN_ID || // TaskCluster, dsari
-    exports2.name || false));
+      (env2.BUILD_ID || // Jenkins, Cloudbees
+        env2.BUILD_NUMBER || // Jenkins, TeamCity
+        env2.CI || // Travis CI, CircleCI, Cirrus CI, Gitlab CI, Appveyor, CodeShip, dsari
+        env2.CI_APP_ID || // Appflow
+        env2.CI_BUILD_ID || // Appflow
+        env2.CI_BUILD_NUMBER || // Appflow
+        env2.CI_NAME || // Codeship and others
+        env2.CONTINUOUS_INTEGRATION || // Travis CI, Cirrus CI
+        env2.RUN_ID || // TaskCluster, dsari
+        exports2.name || false));
     function checkEnv(obj) {
       if (typeof obj === "string") return !!env2[obj];
       if ("env" in obj) {
         return env2[obj.env] && env2[obj.env].includes(obj.includes);
       }
       if ("any" in obj) {
-        return obj.any.some(function(k) {
+        return obj.any.some(function (k) {
           return !!env2[k];
         });
       }
-      return Object.keys(obj).every(function(k) {
+      return Object.keys(obj).every(function (k) {
         return env2[k] === obj[k];
       });
     }
@@ -12887,7 +12887,7 @@ var $ = {
 function init(x, y) {
   let rgx = new RegExp(`\\x1b\\[${y}m`, "g");
   let open = `\x1B[${x}m`, close = `\x1B[${y}m`;
-  return function(txt) {
+  return function (txt) {
     if (!$.enabled || txt == null) return txt;
     return open + (!!~("" + txt).indexOf(close) ? txt.replace(rgx, close + open) : txt) + close;
   };
@@ -13097,7 +13097,7 @@ function LineStream(options) {
   this._lineBuffer = [];
   this._keepEmptyLines = options.keepEmptyLines || false;
   this._lastChunkEndedWithCR = false;
-  this.on("pipe", function(src) {
+  this.on("pipe", function (src) {
     if (!this.encoding) {
       if (src instanceof import_stream.default.Readable) {
         this.encoding = src._readableState.encoding;
@@ -13106,7 +13106,7 @@ function LineStream(options) {
   });
 }
 import_util.default.inherits(LineStream, import_stream.default.Transform);
-LineStream.prototype._transform = function(chunk, encoding, done) {
+LineStream.prototype._transform = function (chunk, encoding, done) {
   encoding = encoding || "utf8";
   if (Buffer.isBuffer(chunk)) {
     if (encoding == "buffer") {
@@ -13129,13 +13129,13 @@ LineStream.prototype._transform = function(chunk, encoding, done) {
   this._lineBuffer = this._lineBuffer.concat(lines);
   this._pushBuffer(encoding, 1, done);
 };
-LineStream.prototype._pushBuffer = function(encoding, keep, done) {
+LineStream.prototype._pushBuffer = function (encoding, keep, done) {
   while (this._lineBuffer.length > keep) {
     const line = this._lineBuffer.shift();
     if (this._keepEmptyLines || line.length > 0) {
       if (!this.push(this._reencode(line, encoding))) {
         const self = this;
-        setImmediate(function() {
+        setImmediate(function () {
           self._pushBuffer(encoding, keep, done);
         });
         return;
@@ -13144,10 +13144,10 @@ LineStream.prototype._pushBuffer = function(encoding, keep, done) {
   }
   done();
 };
-LineStream.prototype._flush = function(done) {
+LineStream.prototype._flush = function (done) {
   this._pushBuffer(this._chunkEncoding, 0, done);
 };
-LineStream.prototype._reencode = function(line, chunkEncoding) {
+LineStream.prototype._reencode = function (line, chunkEncoding) {
   if (this.encoding && this.encoding != chunkEncoding) {
     return Buffer.from(line, chunkEncoding).toString(this.encoding);
   } else if (this.encoding) {
@@ -14746,21 +14746,21 @@ function klona(x) {
   }
   if (str === "[object Array]") {
     k = x.length;
-    for (tmp = Array(k); k--; ) {
+    for (tmp = Array(k); k--;) {
       tmp[k] = klona(x[k]);
     }
     return tmp;
   }
   if (str === "[object Set]") {
     tmp = /* @__PURE__ */ new Set();
-    x.forEach(function(val) {
+    x.forEach(function (val) {
       tmp.add(klona(val));
     });
     return tmp;
   }
   if (str === "[object Map]") {
     tmp = /* @__PURE__ */ new Map();
-    x.forEach(function(val, key) {
+    x.forEach(function (val, key) {
       tmp.set(klona(key), klona(val));
     });
     return tmp;
@@ -17600,23 +17600,23 @@ export class PrismaClient<
   $use(cb: Prisma.Middleware): void
 
 ${[
-      executeRawDefinition.bind(this)(),
-      queryRawDefinition.bind(this)(),
-      batchingTransactionDefinition.bind(this)(),
-      interactiveTransactionDefinition.bind(this)(),
-      runCommandRawDefinition.bind(this)(),
-      metricDefinition.bind(this)(),
-      applyPendingMigrationsDefinition.bind(this)(),
-      this.clientExtensionsDefinitions.prismaClientDefinitions
-    ].filter((d) => d !== null).join("\n").trim()}
+        executeRawDefinition.bind(this)(),
+        queryRawDefinition.bind(this)(),
+        batchingTransactionDefinition.bind(this)(),
+        interactiveTransactionDefinition.bind(this)(),
+        runCommandRawDefinition.bind(this)(),
+        metricDefinition.bind(this)(),
+        applyPendingMigrationsDefinition.bind(this)(),
+        this.clientExtensionsDefinitions.prismaClientDefinitions
+      ].filter((d) => d !== null).join("\n").trim()}
 
     ${(0, import_indent_string7.default)(
-      dmmf.mappings.modelOperations.filter((m) => m.findMany).map((m) => {
-        let methodName = lowerCase(m.model);
-        if (methodName === "constructor") {
-          methodName = '["constructor"]';
-        }
-        return `/**
+        dmmf.mappings.modelOperations.filter((m) => m.findMany).map((m) => {
+          let methodName = lowerCase(m.model);
+          if (methodName === "constructor") {
+            methodName = '["constructor"]';
+          }
+          return `/**
  * \`prisma.${methodName}\`: Exposes CRUD operations for the **${m.model}** model.
   * Example usage:
   * \`\`\`ts
@@ -17625,9 +17625,9 @@ ${[
   * \`\`\`
   */
 get ${methodName}(): Prisma.${m.model}Delegate<ExtArgs>;`;
-      }).join("\n\n"),
-      2
-    )}
+        }).join("\n\n"),
+        2
+      )}
 }`;
   }
   toTS() {
@@ -18166,8 +18166,8 @@ async function buildClient({
   if (generator.previewFeatures.includes("driverAdapters")) {
     fileMap["default.js"] = JS(trampolineTsClient);
     fileMap["default.d.ts"] = TS(trampolineTsClient);
-    fileMap["wasm-worker-loader.js"] = `export default (await import('./query_engine_bg.wasm')).default`;
-    fileMap["wasm-edge-light-loader.js"] = `export default (await import('./query_engine_bg.wasm?module')).default`;
+    fileMap['wasm-worker-loader.js'] = `export { default } from './query_engine_bg.wasm'`
+    fileMap['wasm-edge-light-loader.js'] = `export { default } from './query_engine_bg.wasm?module'`
     pkgJson["browser"] = "default.js";
     pkgJson["imports"] = {
       // when `import('#wasm-engine-loader')` is called, it will be resolved to the correct file
